examples/omni-lock-metamask/index.tsx

```diff
@@ -49,7 +49,10 @@ export function App() {
 
     transfer({ amount: transferAmount, from: omniAddr, to: transferAddr })
       .then(setTxHash)
-      .catch((e) => alert(e.message || JSON.stringify(e)))
+      .catch((e) => {
+        console.log(e);
```

Comment:
remove console

Reply:
I think it's necessary. Because the user may change the example, and put it to console takes more help.